src/app/components/universal/newSafetyDocumentUpdate/newSafetyDocumentUpdate.component.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+
+  width: 80%;
+  height: 80%;
+
+}
+
+@media (max-width: 390px) {
+  .container {
+    width: 90%;
+    height: auto;
+    border: 8px dashed red;
+    border-radius: 3px;
+    margin: 10px auto;
+  }
+  .bi-cone-striped {
+    display: block;
+    margin: 0 auto;
+    color: orange;
+    font-size: 4em;
+  }
+
+  h5 {
```

Comment:
Let's just use classes for these, just in case we add other tags of the same type to the file.

Reply:
fixed to a unique title, and updated html tag to reflect this